bib/cmd/bootc-image-builder/main.go

```diff
@@ -139,9 +140,11 @@ func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest
 	// to using containers storage in all code paths happened.
 	// We might want to change this behaviour in the future to match podman.
 	if !c.Local {
-		pullCmd := exec.Command("podman", "pull", "--arch", c.Architecture.String(), c.Imgref)
+		pullCmd := exec.Command("podman", "pull", "--arch", c.Architecture.String(), fmt.Sprintf("--tls-verify=%v", c.TLSVerify), c.Imgref)
```

Comment:
Doesn't
```
pullCmd := exec.Command("podman", "pull", "--arch", c.Architecture.String(), "--tls-verify", c.TLSVerify, c.Imgref)
```
Work?

Reply:
No, it didn't work for me. It treated `false` and `true` as image refs and tried to pull these.

Reply:
Ok, this is because --tls-verify is legal. where as --tls-verify=false turns it off.